Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```
@@ -229,27 +263,32 @@ configuration:
   hidden:
   - marketplacev2
   required: false
+  section: Collect
 - defaultvalue: 'false'
   display: 'Use system proxy settings'
   name: proxy
   type: 8
   required: false
+  section: Collect
```

```suggestion
  section: Connect
```
proxy is under connect